src/AppBundle/Issues/Status.php

```diff
@@ -27,12 +27,15 @@ final class Status
 
     const BRANCH_80 = '8.0.x';
 
+    const BRANCH_81 = '8.1.x';
+
     const DEVELOP_BRANCH = 'develop';
 
     public static $branches = [
         self::BRANCH_177,
```

Comment:
I'd say this one could be deleted

Reply:
We hesitated as well, but the branch still exists on our repo so we though it could be cleaned later and for now we should focus on the fix for 8.1

Reply:
We also discussed with @lartist about changing the workflow of this tool, for now it uses webhooks to call an action on one of our server It would be easier to handle this if it as via a github action